interpreter/runtests.py

```diff
@@ -30,6 +31,8 @@
 
 class RunTests(unittest.TestCase):
   def _runCommand(self, command, logPath, expectedExitCode = 0):
+    if isWindows:
+      command = command.replace('\'', '"').replace('\\', '/')
```

Comment:
I think the problem was merely insufficient quotation in main.ml. I just pushed a quick fix:

https://github.com/WebAssembly/spec/commit/6aa5c271fd2f5ebbdb9c0a3a63fced0e43ccdcd0

Can you rebase and see whether that makes this line unnecessary? Then the isWindows flag might be unnecessary as well.